python/ray/logger.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+import sys
+
+default_logging_level = logging.INFO
+default_logger = logging.getLogger("ray_default")
```

Comment:
Can you please use `__name__` for logger in Ray? Instead of reusing same logger everywhere? See [here](https://docs.python.org/3/library/logging.html#logger-objects):

```
The name is potentially a period-separated hierarchical value, like foo.bar.baz (though it could also be just plain foo, for example). Loggers that are further down in the hierarchical list are children of loggers higher up in the list. For example, given a logger with a name of foo, loggers with names of foo.bar, foo.bar.baz, and foo.bam are all descendants of foo. The logger name hierarchy is analogous to the Python package hierarchy, and identical to it if you organise your loggers on a per-module basis using the recommended construction logging.getLogger(__name__). That’s because in a module, __name__ is the module’s name in the Python package namespace.
```

This makes it then very easy to configure things for just a subset of Ray logging. For example, I could suppress messages from `ray start` while keeping other messages around.

Reply:
This is a good, thanks.

Reply:
I agree that it'd be better to give each file a logger. 
Currently, Java worker uses one logger object everywhere as well. And I'm planning to change that.